docker-compose.yml

```diff
@@ -2,28 +2,28 @@ version: '3.4'
 services:
   weaviate:
     command:
-      - --host
-      - 0.0.0.0
-      - --port
-      - '8080'
-      - --scheme
-      - http
-      - --read-timeout
-      - '10000s'
-      - --write-timeout
-      - '10000s'
-    image: semitechnologies/weaviate:1.9.0
+    - --host
+    - 0.0.0.0
+    - --port
+    - '8080'
+    - --scheme
+    - http
+    - --read-timeout
+    - '10000s'
+    - --write-timeout
+    - '10000s'
+    image: semitechnologies/weaviate:preview-automatically-request-properties-that-are-needed-for-groupby-e98008c
     restart: on-failure:0
     ports:
-     - "9999:8080"
+     - "8080:8080"
+     - "50051:50051"
     environment:
       QUERY_DEFAULTS_LIMIT: 25
+      QUERY_MAXIMUM_RESULTS: 10000
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
-      PERSISTENCE_DATA_PATH: "./data"
-      DEFAULT_VECTORIZER_MODULE: text2vec-transformers
-      ENABLE_MODULES: text2vec-transformers
+      PERSISTENCE_DATA_PATH: './data'
+      DEFAULT_VECTORIZER_MODULE: 'text2vec-openai'
+      ENABLE_MODULES: 'text2vec-openai,text2vec-cohere,text2vec-huggingface,text2vec-palm,generative-openai,generative-cohere'
```

Comment:
Is it necessary to use text2vec-openai here even after chunking the inpu? Why can't we keep using text2vec-transformers?
Also, I think for this PR only text2vec-openai or text2vec-transformers are required. All other modules are not necessary here?

Reply:
Hey, so even though after chunking the transformers API was running, the container was taking too much space in my system and hence it was being killed by my OS. I tried inserting smaller entries at a time like 50, 100 using "for loops" but that also led to exiting. So I thought that since it is a system issue and not from the code part, I went ahead with OpenAI API.

Reply:
Did you have the same issue before the conversion to the new Python client as well? Because I ran the old demo before and it worked with the text2vec-transformers for me. I will review this solution later today and see if I run into the same issues as you when using the text2vec-transformers module. I'll keep you posted.